t/demo/demo.t

```diff
@@ -0,0 +1,23 @@
+use t::APISIX 'no_plan';
+
+log_level('debug');
```

Comment:
Better not use `debug` by default, it's too verbose to check

Reply:
Done